components/kitchen-sink.tsx

```diff
@@ -235,6 +235,16 @@ export const KitchenSink: React.FC = (): JSX.Element => {
                     />
                 </Card.Content>
             </Card>
+            <Card style={styles.card}>
```

Comment:
Can you add more examples of the Hero component? You can refer to https://brightlayer-ui-components.github.io/react/components/hero/examples for suggestions.